utils/offers/relatedOfferAggrOptions.js

```diff
@@ -0,0 +1,32 @@
+const relatedOfferAggrOptions = (query, params) => {
+  const { authorRole, level, languages, subject, sort = 'rating', skip = 0, limit = 9 } = query
+  const { id } = params
+
+  const match = { _id: { $ne: id } }
```

Comment:
looks a lot like getOffers, can't it be done jsut by putting id in query params?